drivers/adc/ad7124/ad7124.h

```diff
@@ -60,11 +60,17 @@
 #define AD7124_MAX_SETUPS	8
 /* Maximum number of channels */
 #define AD7124_MAX_CHANNELS	16
-/* Device IDs
- * The Device IDs for REV-B of boards need to be updated as 0x07 for ad7124-4
- * and 0x17 for ad7124-8 */
-#define AD7124_4_ID         0x14
-#define AD7124_8_ID	        0x16
+
+/* AD7124-4 Device IDs */
+#define AD7124_4_ID      0x06
+/* Chip ID for new silicon in AD7124-4 */
+#define AD7124_4_NEW_ID  0x07
```

Comment:
ID and NEW_ID naming isn't all that suggestive... what should the user choose? based on what?
I would prefer to have a suggestive naming, related maybe to hardware revision?

Reply:
The user per say is not choosing any ID, the intention is that if he uses a board with new silicon or the old silicon , the chip id read check should not fail.
In terms of the boards. It is only the silicon that has been revised. They will only replace the old AD7124 IC on the board with the new silicon before releasing it.  I am not sure if they will call the newer boards with a new revision number or just replace the rev B boards with new silicon and still call it rev B boards. I will confirm this.

Reply:
I have checked with the Apps Engineer, the new silicon that is redesigned has no name as such and we can keep it as new_id